### PR TITLE
🎉 New Source: Everhour [low-code cdk]

### PR DESCRIPTION
**Pull request by tuanchris**
## What
* Add source Everhour
* https://github.com/airbytehq/airbyte/issues/23305
## How
* Integrate with everhour API https://everhour.docs.apiary.io/#introduction/authentication

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by tuanchris** [comment]
Integration test passing:

**Comment by YowanR** [comment]
@sh4sh @nataliekwong can you please take a look at this PR? 🙏

**Comment by nataliekwong** [comment]
Tagging @natalyjazzviolin

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -503,10 +503,17 @@
...
+  dockerRepository: airbyte/everhour
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.com/integrations/sources/everhour
+  icon: everhour.svg
+  sourceType: api
+  releaseStage: alpha
   allowedHosts:
     hosts:
-      - "${subdomain}.apilayer.com"
-      - apilayer.com
```
Comment (sh4sh) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:509]: Were these removed intentionally? This hosts block was originally under the Exchange Rates API connector definition, above Everhour.

Comment (tuanchris) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:509]: Oh, no. @sh4sh I reverted the change.

**Comment by tuanchris** [comment]
Can you have another look when you have the chance? @sh4sh

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 2]
```diff
@@ -507,6 +507,17 @@
     hosts:
       - "${subdomain}.apilayer.com"
       - apilayer.com
+- name: Everhour
+  sourceDefinitionId: 6babfc42-c734-4ef6-a817-6eca15f0f9b7
+  dockerRepository: airbyte/everhour
```
Comment (marcosmarxm) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:512]: airbyte/source-everhour

File: airbyte-integrations/connectors/source-everhour/source_everhour/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  properties:
+    api_key:
+      type: string
+      title: API Key
+      description: >-
+         Everhour API Key. See the <a
+         href="https://everhour.docs.apiary.io/#introduction/authentication">docs</a>
+         for information on how to generate this key.
+      airbyte_secret: true
+
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-everhour/source_everhour/spec.yaml:17]: EOF

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**